src/shared/components/tab/tab.css

```diff
@@ -5,6 +5,7 @@
   width: 100%;
   margin: 0;
   padding-left: 0;
+  -ms-overflow-style: none;
```

Comment:
this will make the item unscrollable on IE edge, i think, and tabs that have been clipped off will be inaccessible? maybe 
```css
overflow-x: auto; 
```
is what we want?

Reply:
https://github.com/bring/hedwig/pull/782/files#diff-b671d04baa6a7476fba77d923b8e2922R2
we already have `overflow-x: auto;` property. 
with `-ms-overflow-style: none` browser doesn't display the scrollbar and will retain the scroll behavior
https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-overflow-style

Reply:
i believe we have the ```css overflow-x: scroll;``` property, not auto - was not able to scroll when ```-ms-overflow-style: none;``` - Was not able to emulate the scrolling behavior on MS Edge running in virtual box

Reply:
i believe we have the ```overflow-x: scroll;``` property, not auto - Was not able to emulate the scrolling behavior with ```-ms-overflow-style: none;``` on edge running in virtual box

Reply:
yes its `overflow-x: scroll;`. even `overflow-x: auto;` gives a scrollbar when the content overflows.
as per design, the horizontal scroll-bars should be hidden always.
I tested `-ms-overflow-style: none;` in edge and IE11 in Windows 10 and touchpad scroll works.
Broswerstack and VirtualBox don't support horizontal trackpad/mouse scroll that might be the reason why it didn't work. they can be tested with arrow keys

Reply:
ok - good to know that it is a support issue with virtual box 👍

Reply:
Yes, the browserstack doesn't seem to support the horizontal scroll via touchpad, the arrow works, LGTM.